plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -284,6 +284,21 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("list", "watch").Groups(batchGroup).Resources("jobs", "cronjobs").RuleOrDie(),
 			},
 		},
+		{
+			// a role for an external/out-of-tree persistent volume provisioner
+			ObjectMeta: metav1.ObjectMeta{Name: "system:persistent-volume-provisioner"},
+			Rules: []rbac.PolicyRule{
+				rbac.NewRule("get", "list", "watch", "create", "delete").Groups(legacyGroup).Resources("persistentvolumes").RuleOrDie(),
+				// update is needed in addition to read access for setting lock annotations on PVCs
+				rbac.NewRule("get", "list", "watch", "update").Groups(legacyGroup).Resources("persistentvolumeclaims").RuleOrDie(),
+				rbac.NewRule(Read...).Groups(storageGroup).Resources("storageclasses").RuleOrDie(),
```

Comment:
I see here that we're allowing the "system:persistent-volume-provisioner" _ClusterRole_ to read "storageclass" resources. The _ClusterRole_ "edit" allows subjects to create PVCs, which _mention_ these classes.

Should we allow the "edit" _ClusterRole_ to also (get|list|watch) "storageclass" resources, so that they know which classes are available from which to choose?